webui/react/src/components/NavigationSideBar.tsx

```diff
@@ -119,6 +119,7 @@ const NavigationSideBar: React.FC = () => {
 
   const [showSettings, setShowSettings] = useState<boolean>(false);
   const gasLinkOn = useFeature().isOn('genai');
+  const templatesOn = useFeature().isOn('templates');
```

Comment:
**super nit:** not a problem at all to keep this short and simple as `templates`, just wondering if there are other things that are templates in our system that can conflict and bump this to be `taskTemplates`.